pkgs/servers/isso/default.nix

```diff
@@ -1,26 +1,32 @@
-{ stdenv, python2, fetchFromGitHub }:
+{ stdenv, python3Packages, fetchFromGitHub }:
+
+with python3Packages; buildPythonApplication rec {
 
-with python2.pkgs; buildPythonApplication rec {
   pname = "isso";
-  version = "0.12.2";
+  # Can not use 0.12.2 because of:
+  # https://github.com/posativ/isso/issues/617
+  version = "master";
```

Comment:
common convention is to use `unstable-YYYY-MM-DD` where the date is taken from the day of the commit. :)

Reply:
fixed